games-puzzle/kpat/kpat-17.12.1.recipe

```diff
@@ -0,0 +1,139 @@
+	lib:libKF5JobWidgets$secondaryArchSuffix
+	lib:libKF5KDEGames$secondaryArchSuffix
+	lib:libKF5KIOCore$secondaryArchSuffix
+	lib:libKF5NewStuff$secondaryArchSuffix
+	lib:libKF5Service$secondaryArchSuffix
+	lib:libKF5WidgetsAddons$secondaryArchSuffix
+	lib:libKF5XmlGui$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Qml$secondaryArchSuffix
+	lib:libQt5Quick$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	extra_cmake_modules >= 5.39
+	haiku${secondaryArchSuffix}_devel
+	devel:libKF5Attica$secondaryArchSuffix
+	devel:libKF5Auth$secondaryArchSuffix
+	devel:libKF5Bookmarks$secondaryArchSuffix
+	devel:libKF5Codecs$secondaryArchSuffix
+	devel:libKF5Completion$secondaryArchSuffix
+	devel:libKF5ConfigCore$secondaryArchSuffix
+	devel:libKF5ConfigWidgets$secondaryArchSuffix
+	devel:libKF5CoreAddons$secondaryArchSuffix
+	devel:libKF5Crash$secondaryArchSuffix
+	devel:libKF5DBusAddons$secondaryArchSuffix
+	devel:libKF5GuiAddons$secondaryArchSuffix
+	devel:libKF5I18n$secondaryArchSuffix
+	devel:libKF5ItemViews$secondaryArchSuffix
+	devel:libKF5JobWidgets$secondaryArchSuffix
+	devel:libKF5KDEGames$secondaryArchSuffix
+	devel:libKF5KIOCore$secondaryArchSuffix
+	devel:libKF5NewStuff$secondaryArchSuffix
+	devel:libKF5Service$secondaryArchSuffix
+	devel:libKF5Solid$secondaryArchSuffix
+	devel:libKF5WidgetsAddons$secondaryArchSuffix
+	devel:libKF5XmlGui$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Qml$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:gettext$secondaryArchSuffix
+	cmd:make
+	cmd:python2
+	cmd:update_mime_database$secondaryArchSuffix
+	"
+
+BUILD()
+{
+
+	mkdir -p build
+	cd build
+
+	cmake .. \
+		-DECM_DIR=/system/data/cmake/Modules/ECM/cmake \
+		-DCMAKE_INSTALL_PREFIX=$prefix
+		-DKDE_INSTALL_BINDIR=$appsDir/KPatience \
+		-DKDE_INSTALL_LIBDIR=$appsDir/KPatience/lib \
+		-DKDE_INSTALL_DATADIR=$dataDir \
+		-DKDE_INSTALL_DATAROOTDIR=$dataDir \
+		-DKDE_INSTALL_PLUGINDIR=$addOnsDir/Qt5
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	make install
+
+	rm -rf $appsDir/kpat/include \
+		$dataDir/applications \
+		$dataDir/icons \
+		$dataDir/metainfo \
+		$prefix/etc
+
+	local APP_SIGNATURE="application/x-vnd.kde-kpat"
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local LONG_INFO="$SUMMARY"
+	sed \
+		-e "s|@APP_SIGNATURE@|$APP_SIGNATURE|" \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@LONG_INFO@|$LONG_INFO|" \
+		$portDir/additional-files/kpat.rdef.in > kpat.rdef
+
+	addResourcesToBinaries kpat.rdef $appsDir/kpat/kpat
+	addAppDeskbarSymlink $appsDir/kpat/kpat
+}
+
+TEST()
+{
+	make test
```

Comment:
Did you forgot to `cd build`?

Reply:
Does it fail here?, i actually took an existing recipe and modified it, so it was already like that, and it builds just fine as it is locally.

Reply:
`TEST()` is only used if you run `haikuporter --test kpat`.

(BTW, before running this command, you need to have built the package. And if you call `haikuporter -c kpat` then you need to build it again.)

Regading the `cd build`, if it is needed, then you might wish to combine it with the `make test`:
```diff
-	cd build
-	make test
+	make -C build test
```